ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/batch/BatchedSubmissionValidator.scala

```diff
@@ -395,7 +392,7 @@ class BatchedSubmissionValidator[CommitResult] private[validator] (
       commitStrategy: CommitStrategy[CommitResult])(
       implicit executionContext: ExecutionContext): Future[Unit] = {
     val (logEntry, outputState) = logEntryAndState
-    withSubmissionLoggingContext(correlatedSubmission) { implicit loggingContext =>
+    withSubmissionLoggingContext(correlatedSubmission) { _ =>
```

Comment:
With the additional implication that this `withSubmissionLoggingContext` call also does nothing, and can be removed.

Reply:
Yup. This worries me, because it means we're not logging.

@miklos-da, is this supposed to be logging somewhere?

Reply:
Currently this uses a non-contextual logger but I believe this is preparation for contextual logging where the context is the correlation ID; if this is not the case (or if the usage of contextual logging is not coming shortly) then I'd remove it for now. @miklos-da can probably elaborate.

Reply:
Also considered in sandbox: https://github.com/digital-asset/daml/pull/7035/files#r465983041

Reply:
We just used this bit to add a correlation ID to all log events transparently, i.e., not requiring a previous context. Threading a logging context through the stack is to be done later.